pkg/cmd/server/origin/master.go

```diff
@@ -33,6 +33,12 @@ import (
 	"github.com/openshift/origin/pkg/user/cache"
 )
 
+const (
```

Comment:
in later cleanup, we may move these to the oauth/apiserver package.  We just don't really want people depending on the package to get a constant through.